currency_rate_update_boi/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,9 @@
+**Italiano**
+
+Suggeriamo di eseguire l'azione pianificata "Currency Rates Update" alla fine del giorno,
+per essere sicuri che la Banca d'Italia abbia aggiornato i dati.
+
+**English**
+
+We suggest to execute the "Currency Rates Update" cron at the end of the day,
```

Comment:
L'azione in questione dovrebbe essere "Currency Rates Update (OCA) daily".
Che sia il caso di mettere il nome completo?
Non vorrei ci fosse poi confusione con altri moduli che introducono azioni simili. 
Il dubbio mi sorge perché vedo che è stato specificato OCA nel nome.

Reply:
Messo "Currency Rates Update (OCA) daily"